pycbc/workflow/coincidence.py

```diff
@@ -779,14 +818,35 @@ def setup_multiifo_combine_statmap(workflow, final_bg_file_list, bg_file_list,
 
     if cstat_exe_name == 'pycbc_multiifo_combine_statmap':
         combine_statmap_node = cstat_exe.create_node(final_bg_file_list,
-                                                     tags)
+                                                     tags=tags)
     elif cstat_exe_name == 'pycbc_multiifo_add_statmap':
         combine_statmap_node = cstat_exe.create_node(final_bg_file_list,
                                                      bg_file_list,
-                                                     tags)
+                                                     tags=tags)
+
     workflow.add_node(combine_statmap_node)
     return combine_statmap_node.output_file
 
+def setup_multiifo_exclude_zerolag(workflow, statmap_file, other_statmap_files,
+                                   out_dir, tags=None):
+    """
+    Exclude single triggers close to zerolag triggers from forming any
+    background events
+    """
+    if tags is None:
+        tags = []
+    make_analysis_dir(out_dir)
+    logging.info('Setting up multiifo exclude zerolag')
+
+    exc_zerolag_exe = PyCBCMultiifoExcludeZerolag(workflow.cp, 'exclude_zerolag',
+                                                  ifos=workflow.ifos, tags=tags,
+                                                  out_dir=out_dir)
+    exc_zerolag_node = exc_zerolag_exe.create_node(statmap_file,
+                                                   other_statmap_files,
+                                                   tags=tags)
```

Comment:
So *don't* send them again here! This would be for tags that vary on the node level (which in this case is none of them).

Reply:
Thanks for that, it doesn't repeat the tags which is all I was wanting.

One thing I am a bit confused by though is the naming of the file: the `exclude_foreground` files all start with H1L1V1, even when its a 2-detector output. However the `multiifo_statmap` ones all only have their used detectors. Is this something easy to change or even needed?

Examples:
`H1L1V1-EXCLUDE_ZEROLAG_FULL_2DET_H1V1-1185937218-687600.hdf`
`L1H1-MULTIIFO_STATMAP_FULL_DATA_FULL_2DET_H1L1_VETOES-1185937218-687600.hdf`

Reply:
You can choose that. When initializing the executable you choose the set of ifos that are active. In many cases this is `workflow.ifos` in some cases this is a specific subset.

Reply:
Thanks! This is in now